powerauth-tpp-engine/src/main/resources/application.properties

```diff
@@ -0,0 +1,38 @@
+# Allow externalization of properties using application-ext.properties
+spring.profiles.active=ext
+
+# Database Configuration - MySQL
+spring.datasource.url=jdbc:mysql://localhost:3306/powerauth
+spring.datasource.username=powerauth
+spring.datasource.password=
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+spring.jpa.properties.hibernate.connection.characterEncoding=utf8
+spring.jpa.properties.hibernate.connection.useUnicode=true
+spring.jpa.hibernate.use-new-id-generator-mappings=false
```

Comment:
I would move this property to the bottom of this file, otherwise someone may comment it out together with MySQL properties.